HACKING.md

```diff
@@ -0,0 +1,120 @@
+Flatpak uses a traditional autoconf-style build mechanism. The exact steps
```

Comment:
If you rename this file to `CONTRIBUTING.md`, Github (and Gitlab) will automatically use it as a contributor guide.

Reply:
Hmm, it's helpful to potential contributors but I'm not sure it's really a guide to contributing? The docs say...

> In the new file, add contribution guidelines. These could include:
> Steps for creating good issues or pull requests.
> Links to external documentation, mailing lists, or a code of conduct.
> Community and behavioral expectations.

Reply:
Or maybe I'm just splitting hairs

Reply:
I went ahead and renamed it.